cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -34,7 +34,9 @@
 {# Overlay for the page. Used for the mobile mega menu. #}
 <div class="a-overlay u-hidden"></div>
 
-<a href="#main" id="skip-nav">Skip to main content</a>
+<div class="skip-nav">
+    <a href="#main" class="skip-nav_link">Skip to main content</a>
+</div>
```

Comment:
Why convert this to `div > a` instead of just leaving it as an `a`?

Reply:
As discussed in chat, this is because the skip link is absolutely positioned, but needs a container.

Added a modifier class and increased the margin, so now it looks like:

<img width="375" alt="screen shot 2018-11-26 at 1 06 41 pm" src="https://user-images.githubusercontent.com/704760/49033140-61920b80-f17c-11e8-9f8d-e3ebb0a26da0.png">

<img width="405" alt="screen shot 2018-11-26 at 1 06 36 pm" src="https://user-images.githubusercontent.com/704760/49033139-61920b80-f17c-11e8-9962-0a37297cb809.png">